### PR TITLE
Changed fault LED red colour

### PR DESCRIPTION
**Pull request by isaachilly**
### Description of work

Changed the hue of the fault RED colour for the LEDs in Fermichopper. 

### Ticket

https://github.com/ISISComputingGroup/IBEX/issues/7863

### Acceptance criteria

Fermichopper fault lights are a clearer/brighter RED than before.

### Unit tests

*Give an overview of unit tests you have added or modified, if applicable. The aim is provide information to help the reviewer*

### System tests

*Mention any automated tests or manual tests that you have added or modified, if applicable. The aim is provide information to help the reviewer*

### Documentation
*Highlight and provide a link to any additions or changes to the documentation, if applicable. The aim is provide information to help the reviewer*

---

#### Code Review

- [x] Is the code of an acceptable quality?
- [x] If the change is to an OPI, does the `check_opi_format.py` script in [C:\Instrument\Dev\ibex_gui\base\uk.ac.stfc.isis.ibex.opis](https://github.com/ISISComputingGroup/ibex_gui/blob/master/base/uk.ac.stfc.isis.ibex.opis/check_opi_format.py) pass?
- [x] Do the changes function as described and is it robust?
- [x] Is there associated PR for the [release notes](https://github.com/ISISComputingGroup/IBEX/blob/master/release_notes/ReleaseNotes_Upcoming.md)?

### Final Steps
- [ ] Reviewer has also merged the [release notes](https://github.com/ISISComputingGroup/IBEX/blob/master/release_notes/ReleaseNotes_Upcoming.md)

### CODE REVIEW

File: base/uk.ac.stfc.isis.ibex.opis/resources/fermi_chopper.opi [hunk 1]
```diff
@@ -2265,27 +2265,27 @@ $(pv_value)</tooltip>
...
           <height>25</height>
           <name>LED</name>
           <off_color>
-            <color name="ISIS_Red_LED_Off" red="102" green="0" blue="0"/>
+            <color name="ISIS_Red_LED_Off" red="102" green="0" blue="0" />
           </off_color>
           <off_label>OFF</off_label>
           <on_color>
-            <color name="ISIS_Red_LED_On" red="255" green="100" blue="100"/>
+            <color name="ISIS_Emergency_Button_Background" red="255" green="0" blue="0" />
```
Comment (will2201) [inline base/uk.ac.stfc.isis.ibex.opis/resources/fermi_chopper.opi:2277]: (First instance) Still appropriate to use the **ISIS_Red_LED_On** name for the <color> tag to reflect what the LED does.

**Review by will2201 — CHANGES_REQUESTED** [review]
The RGB values have been updated correctly. But the "name" attribute on all of the <color> tags that have been changed should still be "ISIS_Red_LED_On".

File: base/uk.ac.stfc.isis.ibex.opis/resources/fermi_chopper.opi (hunk 1 above)
Comment (isaachilly) [reply base/uk.ac.stfc.isis.ibex.opis/resources/fermi_chopper.opi]: Made the change

**Review by will2201 — APPROVED** [review]
LGTM

**Pull request merged**